compiler/noirc_frontend/src/hir/resolution/resolver.rs

```diff
@@ -101,6 +101,9 @@ pub struct Resolver<'a> {
     /// for these so we can still resolve them in the parent module without them being in a contract.
     in_contract: bool,
 
+    /// True if the current crate is the Noir standard library.
+    in_stdlib: bool,
```

Comment:
One more nit. We have access to the `PathResolver` in `Resolver`. You should be able to fetch the module id using `module_id()` which has a crate ID attached. You can then call `is_stdlib()` on the crate.

Reply:
nice catch, made the change now.